Readme.md

```diff
@@ -132,15 +139,20 @@ see where it's stored) if you don't plan to reinstall Dolphin.
 * -V, --video_backend=<str> Specify a video backend
 * -A, --audio_emulation=<str> Low level (LLE) or high level (HLE) audio
 
-Available DSP emulation engines are HLE (High Level Emulation) and
-LLE (Low Level Emulation). HLE is fast but often less accurate while LLE is
-slow but close to perfect. Note that LLE has two submodes (Interpreter and
-Recompiler), which cannot be selected from the command line.
+When the `-d` argument is used, debugger features can be viewed from the "View" menu.
```

Comment:
I'm not sure how useful mentioning this is. The option is documented in the command line --help output.

Also, the option directly shows the debugger pane. You don't need to use the View menu to make it appear.